### PR TITLE
👓 Proofreading?

### PR DESCRIPTION
**Pull request by greymistcube**
👓

### CODE REVIEW

File: Libplanet.Common/Libplanet.Common.csproj [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
-      <PackageReference Include="Bencodex" Version="0.10.0" />
-      <PackageReference Include="Bencodex.Json" Version="0.10.0" />
-      <PackageReference Include="BouncyCastle.Cryptography" Version="2.0.0" />
-      <PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
-      <PackageReference Include="System.Collections.Immutable" Version="1.7.1" />
-      <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
-      <PackageReference Include="System.IO" Version="4.3.0" />
-      <PackageReference Include="System.Linq" Version="4.3.0" />
-      <PackageReference Include="System.Text.Json" Version="6.0.7" />
-    </ItemGroup>
```
Comment (riemannulus) [inline Libplanet.Common/Libplanet.Common.csproj:30]: Is it intended?

Comment (greymistcube) [reply Libplanet.Common/Libplanet.Common.csproj:30]: Yes? Partially moved down to a different place. Also, unless there is a good reason to use a specific `System` package versions, I don't think we should include them in the list.

Comment (riemannulus) [reply Libplanet.Common/Libplanet.Common.csproj:30]: But if remove those packages, cannot build `Libplanet.Common`. Have you tried rebuilding with this PR?

Comment (riemannulus) [reply Libplanet.Common/Libplanet.Common.csproj:30]: My mistake. 😓 Build well.

**Review by riemannulus — APPROVED** [review]

**Pull request merged**